addons/tooltips/addon.json

```diff
@@ -0,0 +1,116 @@
+{
+  "name": "Tooltips",
```

Comment:
I agree with @mxmou, this should be `Link tooltips`

or hovercards, but then the description should probably change `tooltips` to `hovercards` too

Reply:
If you haven't done with that go "unresolve" that and continue there